syntax = "proto3";

import "Ack.proto";
message CreateSessionAck {
    optional Ack.AckStatus status = 1;
    optional string message = 2;
    optional int32 session_id = 3;
}